.manylinux-install.sh

```diff
@@ -23,15 +23,20 @@ fi
 ls -ld /cache
 ls -ld /cache/pip
 
+# We need some libraries because we build wheels from scratch:
+yum -y install libffi-devel
+
 # Compile wheels
 for PYBIN in /opt/python/*/bin; do
-    if [[ "${PYBIN}" == *"cp27"* ]] || \
+    if \
+       [[ "${PYBIN}" == *"cp27"* ]] || \
        [[ "${PYBIN}" == *"cp35"* ]] || \
+       [[ "${PYBIN}" == *"cp311"* ]] || \
        [[ "${PYBIN}" == *"cp36"* ]] || \
        [[ "${PYBIN}" == *"cp37"* ]] || \
        [[ "${PYBIN}" == *"cp38"* ]] || \
-       [[ "${PYBIN}" == *"cp310"* ]] || \
-       [[ "${PYBIN}" == *"cp39"* ]]; then
+       [[ "${PYBIN}" == *"cp39"* ]] || \
+       [[ "${PYBIN}" == *"cp310"* ]] ; then
```

Comment:
Why no "cp311"?

Reply:
It gets added in line 34. The Python versions are not in order to make it easier to add specific ones via configuration, see https://github.com/zopefoundation/meta/blob/fba6d957ba447b6fa369d872e803756bd5176391/config/c-code/manylinux-install.sh.j2#L28-L41 for the code which produces this file.